src/analysisd/rules.c

```diff
@@ -296,14 +297,14 @@ int Rules_OP_ReadRules(const char *rulefile, RuleNode **r_node, ListNode **l_nod
             }
 
             if (strcasecmp(rule[j]->element, xml_rule) != 0) {
-                merror("Invalid configuration. '%s' is not "
+                smerror(&tmp_msg, "Invalid configuration. '%s' is not "
```

Comment:
`smerror(&tmp_msg, "Invalid configuration. '%s' is not a valid element.", rule[j]->element);`